paket.dependencies

```diff
@@ -38,6 +38,8 @@ nuget Topshelf.FSharp
 nuget fszmq
 nuget ApacheThrift
 nuget elmah.io.client
+nuget Google.Logging.V2 1.0.0-beta02
```

Comment:
Thanks for picking this up! Was the older version on purpose? Current version is beta04. https://www.nuget.org/packages/Google.Logging.V2/ I think beta05 is due out soon.